modules/backup-and-restore/pages/configurations.adoc

```diff
@@ -32,6 +32,11 @@ Required if backup is to be stored on S3.
 *NOTE*: If setting this in interactive mode, store the key in a file and provide the path to the file, e.g., `@/tmp/test_secret`.
 |`+nan+`
 
+|System.Backup.S3.RoleARN |The AWS role for accessing s3 bucket, its use is prioritized over the combo of access key id and secret access key in accessing s3. Note this is only for AWS S3, and that using role ARN assumes access key id and secret access key are already configured with AWS CLI
```

Comment:
"role ARN assumes access key id and secret access key" is not a must. we only need the cluster nodes to be allowed to assume that role.  there can be several ways,  refer here https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/

Sorry, I didn't catch this in the PR review. If this is confirmed, we also need to update the description of `System.Backup.S3.RoleARN` in the code.

so, how about this one:
 "Note this is only for AWS S3, and it's crucial to ensure that your nodes are trusted and permitted to assume this RoleARN. To verify this, you can use the `aws sts assume-role` command."

feel free to make it better.

Reply:
I've changed the description based on our discussion offline.